lib/presenters/artefact_presenter.rb

```diff
@@ -95,6 +95,10 @@ def present
       }
     end
 
+    if downtime = Downtime.for(@artefact)
+      presented['downtime_message'] = downtime.message if downtime.publicise?
+    end
```

Comment:
for the current use case, we only need a message to be displayed when the downtime needs to be publicised. also i am keen on keeping frontend's responsibility as simple as possible. it shouldn't be difficult to add more stuff as and when needed.